src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -194,7 +196,19 @@ export default {
 		},
 		onSubmit() {
 			alert(this.message)
-		}
+		},
+		showSlashCommands() {
+			this.$refs.contenteditable.showTributeManually('/')
+			// Add an event listener to the document
+			document.addEventListener('click', this.hideTribute)
+		},
+		hideTribute(event) {
+			if (!event.target.closest('.show-slash-button')) {
```

Comment:
We need a click outside of tribute container here, right?
```suggestion
			if (!event.target.closest('#' + this.$refs.contenteditable.tributeId)) {
```

Reply:
No, the tribute should close when clicking inside the tribute too, the only click that would be ignored is the click on the slash command button

Reply:
If you pick something via tribute, it should automatically update text field with selection and close itself, so shouldn't be a problem. I only noticed, that if you won't stop a click event and put a listener, it will be propagated and immediately close the tribute you just opened. So it's not only about ignoring the slash button